pkg/util/utils.go

```diff
@@ -384,11 +388,19 @@ func truncateNameOrIDHash(data string) string {
 	return Sha1WithBase62(data)[:common.Base62HashLength]
 }
 
+func truncateUIDAsSuffix(uid string) string {
```

Comment:
truncateUIDAsSuffix -> truncateUIDHash might be more appropriate.

Reply:
updated.